src/searches.py

```diff
@@ -80,8 +80,10 @@ def getGoogleTrends(self, wordsCount: int) -> list[str]:
 
     def getRelatedTerms(self, term: str) -> list[str]:
         # Function to retrieve related terms from Bing API
-        relatedTerms: list[str] = requests.get(f"https://api.bing.com/osjson.aspx?query={term}",
-                             headers={"User-agent": self.browser.userAgent}, ).json()[1]
+        relatedTerms: list[str] = requests.get(
+            f"https://api.bing.com/osjson.aspx?query={term}",
```

Comment:
Shouldn't this use "SEARCH_URL" or similar?

Reply:
SEARCH_URL's https://bing.com. Might be worth adding a constant for this url but AFAIK it isn't used too much.